index.js

```diff
@@ -85,7 +89,7 @@ exports = module.exports = new Hbs();
  * expose method to create additional instances of `Hbs`
  */
 
-exports.create = function() {
+exports.create = function () {
```

Comment:
there's a bunch of other instances where it looks like your editor modified spacing. please clean those up